aroquev/src/main/webapp/script.js

```diff
@@ -31,45 +30,61 @@ function addRandomGreeting() {
  * Adds Armandos age to the page.
  */
 function calculateArmandoAge() {
-    // Calculate age, Armando was born on May 13, 2000.
-    const diffMs = new Date() - new Date(2000, 5, 13);
-    const ageDt = new Date(diffMs); 
-    
-    // Add it to the page.
-    const ageContainer = document.getElementById('armando-age-container');
-    ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
-    
+  // Calculate age, Armando was born on May 13, 2000.
+  const diffMs = new Date() - new Date(2000, 5, 13);
+  const ageDt = new Date(diffMs); 
+  
+  // Add it to the page.
+  const ageContainer = document.getElementById('armando-age-container');
+  ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);   
 }
 
+calculateArmandoAge();
+
 // Scripts for the image gallery
 var slideIndex = 1;
 
 function initSlides() {
   showSlides(slideIndex);
 }
 
-// Next/previous controls
-function plusSlides(n) {
-  showSlides(slideIndex += n);
+// Next control
+function showNextSlide() {
+	slideIndex ++;
+	showSlides(slideIndex);
+}
+
+// Previous control
+function showPreviousSlide() {
+	slideIndex --;
+	showSlides(slideIndex);
 }
 
 // Thumbnail image controls
-function currentSlide(n) {
-  showSlides(slideIndex = n);
+function updateCurrentSlide(n) {
+  slideIndex = n;
+  showSlides(slideIndex);
 }
 
 function showSlides(n) {
-  var i;
   var slides = document.getElementsByClassName("mySlides");
   var dots = document.getElementsByClassName("dot");
-  if (n > slides.length) {slideIndex = 1}
-  if (n < 1) {slideIndex = slides.length}
-  for (i = 0; i < slides.length; i++) {
-      slides[i].style.display = "none";
+  if (n > slides.length) {
+    slideIndex = 1;
   }
-  for (i = 0; i < dots.length; i++) {
-      dots[i].className = dots[i].className.replace(" active", "");
+  if (n < 1) {
+    slideIndex = slides.length;
```

Comment:
nit: There is a cute way to do these two "if" checks using the "modulo" operator (%)

Reply:
True! Done.